autotests/testkeyverification.h

```diff
@@ -0,0 +1,18 @@
+// SPDX-FileCopyrightText: 2022 Tobias Fella <fella@posteo.de>
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+#pragma once
+
+#include <QTest>
+#include "testutils.h"
+
+class TestKeyVerificationSession : public QObject
+{
+    Q_OBJECT
+
+private Q_SLOTS:
+    void testVerification();
+
+};
+
```

Comment:
You don't need to add a separate `.h` file here; instead, just define the class in `testkeyverification.cpp` and `#include "testverification.moc"` at the end of it (moc will auto-generate it) - see how it's done in `quotest.cpp`.